portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -12,7 +12,7 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text;");
+    response.setContentType("application/json;");
```

Comment:
Did you mean to change this to JSON? Looks like this still returns text?

Reply:
You are correct this should still return text.